src/components/CompanyCard/CompanyCard.tsx

```diff
@@ -14,10 +14,15 @@ const CompanyCard = (props: Props) => {
         className="card-img-top p-3 rounded"
         alt="company__img"
         loading="lazy"
+        data-testid="company-img"
```

Comment:
Please not use data-testis attribute. Check other position

Reply:
Fixed